sdk/storage/Azure.Storage.Common/src/Shared/SasQueryParametersExtensions.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Text;
+
+namespace Azure.Storage.Sas
+{
+    internal static class SasQueryParametersExtensions
```

Comment:
Nit - should we move this all `SasExtensions`?

Reply:
Fixed